esbuild.js

```diff
@@ -0,0 +1,48 @@
+const svgrPlugin = require('esbuild-plugin-svgr');
+const esbuild = require('esbuild');
+const plugin = require('node-stdlib-browser/helpers/esbuild/plugin');
+const stdLibBrowser = require('node-stdlib-browser');
+const { nodeExternalsPlugin } = require('esbuild-node-externals');
+const fs = require('fs');
+
+const entryPoints = fs
```

Comment:
maybe it would be a better approach to use something like [glob](https://www.npmjs.com/package/glob) for implementing the pattern matching?